src/shipit_code_coverage/shipit_code_coverage/notifier.py

```diff
@@ -48,7 +50,7 @@ def notify(self):
                     continue
 
                 if coverage['commit_covered'] < 0.2 * coverage['commit_added']:
-                    content += '* [%s](https://firefox-code-coverage.herokuapp.com/#/changeset/%s): %d covered out of %d added.\n' % (rev, rev, coverage['commit_covered'], coverage['commit_added'])  # noqa
+                    content += '* [%s](https://firefox-code-coverage.herokuapp.com/#/changeset/%s): %d covered out of %d added.\n' % (desc, rev, coverage['commit_covered'], coverage['commit_added'])  # noqa
```

Comment:
Interesting. Could you please share a screenshot of what this message looks like out in the wild?

Reply:
Here's what it looks like currently:
![mail_screen](https://user-images.githubusercontent.com/1616846/36538883-4859c634-17d5-11e8-9dfe-d1e2b4cb673d.png)

After this, the hash will be replaced with the first line of the commit message (e.g. `Bug 107264 - Make prefs parser errors louder. r=glandium` in this case).